### PR TITLE
Add functionality to ignore languages and files from the build

### PR DESCRIPTION
**Pull request by pettinarip**
## Description
Added two **optional** env vars: `GATSBY_BUILD_LANGS` and `IGNORE_CONTENT`, to optimize the build times when running `yarn start` and improve the DX overall.

It mainly focuses on being used only in development mode.

E.g.
Using this config
```sh
GATSBY_BUILD_LANGS=en
IGNORE_CONTENT=**/docs,**/tutorials
```
Build times:
- before 	~4min
- now	~1min

### CODE REVIEW

**Comment by samajammin** [comment]
Awesome! Been wishing for this for a long time 🤩 

Tested locally:

With config:
```
GATSBY_BUILD_LANGS=en
IGNORE_CONTENT=**/docs,**/tutorials
```
`netlify dev` booted up in 1:22.

Running normally, `netlify dev` booted up in 4:14.

Nice work 🎉

**Review by samajammin — APPROVED** [review]
🤩

**Comment by wackerow** [comment]
Hell yeah, `yarn start` just a spun up in less than a minute for me. 💪🏻

**Review by wackerow — APPROVED** [review]

File: gatsby-config.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+
 const defaultLanguage = `en`
 const siteUrl = `https://ethereum.org`
 
+const ignoreContent = (process.env.IGNORE_CONTENT || "")
+  .split(",")
+  .filter(Boolean)
+
+const ignoreTranslations = Object.keys(allLanguages)
+  .filter((lang) => {
```
Comment (samajammin) [inline gatsby-config.js:13]: This could probably be a one-liner 😄

Comment (pettinarip) [reply gatsby-config.js:13]: Oh yes, doing!

**Pull request merged**